setup.py

```diff
@@ -366,6 +372,7 @@ def run(self):
         cmake_cache_vars = defaultdict(lambda: False, cmake.get_cmake_cache_variables())
         if cmake_cache_vars['USE_NUMPY']:
             report('-- Building with NumPy bindings')
+            install_requires += ['numpy']
```

Comment:
This is a bit counterintuitive, because by the time we have `cmake_cache_vars['USE_NUMPY']`, we already know numpy is installed. But this is probably something needed for packaging that's why we have this line.

```suggestion
            global install_requires
            install_requires += ['numpy']
```

Reply:
Yeah, it's needed for packaging. Although the build environment has numpy, that doesn't mean the install environment has numpy. If we build against numpy, you're required to provide numpy in the install environment too.